Content.Shared/Hands/SharedHandsSystem.cs

```diff
@@ -143,9 +143,6 @@ protected virtual void HandleContainerModified(EntityUid uid, SharedHandsCompone
 
         private void HandleContainerInserted(EntityUid uid, SharedHandsComponent component, EntInsertedIntoContainerMessage args)
         {
-            // un-rotate entities. needed for things like directional flashlights
-            Transform(args.Entity).LocalRotation = 0;
-
```

Comment:
removed?

Reply:
probably the result of lazy master merging, the PR is pretty old. I'll go through the diff and clean up unintended changes including this

Reply:
ah never mind, I got confused.

This change was added in #6740, but then a separate engine container change made it redundant, as any container insertion just resets the angle now. So not really related to this PR, but these lines of code just aren't required anymore.